src/Components/Quizbox/Quizbox.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import styles from './Quizbox.module.css';
+
+const Quizbox = ({children}) => {
+    return (
+        <div className={styles.quizbox_container}>
```

Comment:
Kunnen we hier ook semantische elementen voor gebruiken? We gaan richting een `<div>`-jes hell namelijk 🤓 
<img width="733" alt="Screenshot 2021-09-17 at 16 59 48" src="https://user-images.githubusercontent.com/32450174/133804779-05bc1af3-174d-426d-9204-db293570da62.png">